ctx.go

```diff
@@ -77,6 +77,12 @@ type ContextWithBody[B any] interface {
 	Header(key string) string                 // Get request header
 	SetHeader(key, value string)              // Sets response header
 
+	// Returns the underlying net/http, gin or echo context.
+	//
+	// Usage:
+	//  ctx := c.Context() // net/http: the [context.Context] of the *http.Request
+	//  ctx := c.Context().(*gin.Context) // gin: Safe because the underlying context is always a [gin.Context]
+	//  ctx := c.Context().(echo.Context) // echo: Safe because the underlying context is always a [echo.Context]
```

Comment:
```suggestion
	//  ctx := c.Context().(*echo.Context) // echo: Safe because the underlying context is always a [echo.Context]
```

Maybe? Can't tell honestly.

Reply:
No, it's an interface, like Fuego now